rstudio/rstudio
# Disambiguate labels for console focus commands

- Fixes #6616
- Sorry for all the stripped whitespace; let me know if you'd like me to resubmit without that (but easily to hide in GitHub review).

### Intent

We have two commands with the same visible label (i.e. in the Command Palette): "Move Focus To Console".

These are actually different commands; one corresponds to the default <kbd>Ctrl+2</kbd> behavior, the other is a command that is not shown in the menus and has no shortcut, but is used internally. It actually moves focus to the Console **Panel**, the quadrant containing the console, but not necessarily the console itself. It also has the quirk of moving one tab to the right in the console panel.

### Approach

Change the label to "Move Focus to Console Panel" to distinguish from the other command. Left the quirky behavior as-is, as quite likely _something_ is relying on it. Maybe this command shouldn't even be a command but just an internal event?

### QA Notes

Confirm that the command's have different labels in the command palette, e.g.:

<img width="604" alt="screenshot of command palette with focus console search results" src="https://user-images.githubusercontent.com/10569626/90809625-9d68ef00-e2d6-11ea-845b-e41954879383.png">
